projects/demo/src/app/wizard/wizard-basic.demo.html

```diff
@@ -199,11 +203,13 @@
     </p>
   </clr-wizard-page>
 
-  <clr-wizard-page>
-    <ng-template clrPageTitle>Title for page 3</ng-template>
-    <!-- mandatory -->
-    <ng-template clrPageNavTitle>Step 3</ng-template>
-    <!-- optional -->
-    <p class="content-for-page-3">Content for step 3</p>
-  </clr-wizard-page>
+  <ng-container *ngIf="scrollingSteps">
+    <clr-wizard-page
+      *ngFor="let number of [3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30]"
```

Comment:
Can you create a variable or a getter that can simply return 3 or 30 steps depending on scrollingSteps?
```suggestion
      *ngFor="let page of pages"
```

Reply:
Well, it would always return the same array because this is only rendered when "scrolling steps" is enabled. The first two pages are static with different content.

Reply:
Yeah, but now we have 2 of the same pages that have a lot of paragraphs that are the same and these here so we can just use on NgFor for all pages instead of having static and dynamic ones and just change the number when needed.  It will save a lot of space and unecessary code on that page.

Reply:
Okay, I refactored this.